### PR TITLE
updated npmignore file

### PR DESCRIPTION
**Pull request by clickclickonsal**
### Proposed changes
 - Updated .npmignore file

Link to the issue:
https://github.com/marionettejs/backbone.marionette/issues/3142

### CODE REVIEW

**Comment by scott-w** [comment]
Thanks for this @clickclickonsal 👍

File: .npmignore [hunk 1]
```diff
@@ -7,12 +6,16 @@ src
...
+.gitignore
 .travis.yml
 bower.json
 CONTRIBUTING.md
 gulpfile.babel.js
+ISSUE_TEMPLATE.md
+marionette-logo.png
+PULL_REQUEST_TEMPLATE.md
+trigger-deploy-mn-com.js
 upgradeGuide.md
```
Comment (denar90) [inline .npmignore]: I think `changelog.md` is missed.

Comment (clickclickonsal) [inline .npmignore]: I found this in the  [NPM documentation](https://docs.npmjs.com/misc/developers).
image

Comment (denar90) [inline .npmignore]: 👍  then)

**Pull request merged**